mev_inspect/arbitrages.py

```diff
@@ -52,6 +52,11 @@ def _get_arbitrages_from_swaps(swaps: List[Swap]) -> List[Arbitrage]:
         ]
         routes = _get_all_routes(start, end, potential_intermediate_swaps)
 
+        error = ""
```

Comment:
None instead of empty string

Then if it's any string (is not None) we know it's a valid error